SignInCheckIn/SignInCheckIn.Tests/Services/ChildSigninServiceTest.cs

```diff
@@ -803,10 +803,10 @@ public void CheckAcEventDuringCurrentService()
             // Arrange
             _applicationConfiguration.Setup(m => m.AdventureClubEventTypeId).Returns(20);
 
-            DateTime currentMpEventDtoStartTime = new DateTime(2016, 12, 1, 9, 0, 0);
-            DateTime futureMpEventDtoStartTime = new DateTime(2016, 12, 1, 11, 0, 0);
+            var currentMpEventDtoStartTime = new DateTime(2016, 12, 1, 9, 0, 0);
```

Comment:
I'm concerned about using var for all the variable definitions in unit tests - I noticed an issue a couple of sprints ago where the Mock or NUnit List type was being using in place of System...List, which caused problems when I added a type comparison to the test.